src/celeritas/mat/MaterialParams.cc

```diff
@@ -89,18 +111,16 @@ MaterialParams::from_import(ImportData const& data)
 
         for (auto const& elem_comp : material.elements)
         {
-            ElementId elem_def_id{elem_comp.element_id};
-
             // Populate MaterialParams number fractions
             material_params.elements_fractions.push_back(
-                {elem_def_id, elem_comp.number_fraction});
+                {static_cast<ElementId>(elem_comp.element_id),
+                 elem_comp.number_fraction});
         }
-        input.materials.push_back(material_params);
+        input.materials.push_back(std::move(material_params));
     }
 
-    // Construct MaterialParams and return it as a shared_ptr
-    MaterialParams materials(input);
-    return std::make_shared<MaterialParams>(std::move(materials));
+    // Return a MaterialParams shared_ptr
+    return std::make_shared<MaterialParams>(MaterialParams(input));
```

Comment:
Not sure why that was written as before... this should just be:
```suggestion
    return std::make_shared<MaterialParams>(std::move(input));
```